packages/hooks/src/function.ts

```diff
@@ -68,5 +68,16 @@ export const functionHooks = <F, T = any>(original: F, opts: HookSettings<T>) =>
   registerContextUpdater(wrapper, updateContext);
   registerMiddleware(wrapper, middleware);
 
+  const originalProps = (Object.getOwnPropertyNames(original) as any)
```

Comment:
I think this might cause the same issues as https://github.com/daffl/uberproto/issues/25 in some environments.

Reply:
`Object.keys` should work as expected (concat with `Object.getOwnPropertySymbols` is also ok)